Amigolden/src/app/pages/login/login.page.html

```diff
@@ -1,5 +1,36 @@
 <ion-content padding class="login-background no-scroll">
-	<h3 class="login-tag">Unleash yourself</h3>
+	<!-- <h3 class="login-tag">Unleash yourself</h3> -->
+	<ion-grid class="display-flax">
+		<ion-row class="ion-align-items-center flex-grow-1">
+			<ion-col size="12" class="ion-padding" *ngIf="form">
+				<form [formGroup]="form" (ngSubmit)="submit()">
```

Comment:
whats going on with the group here? its point to the old form

Reply:
we need to submit the form, the submit() function currently just logs the data

Reply:
I dont see the method="post" or the login url here.

Reply:
The old form is still at the in the footer it has a #form which will interfere with the form that you define